### PR TITLE
[7.3] PHP7.3 integration tests

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/Fixtures/Integration/misc/PHP7_3.test [hunk 1]
```diff
@@ -103,3 +112,10 @@ rand($a, $b, ); // `random_api_migration` rule
 settype($foo, "integer", ); // `set_type_to_cast` rule
 in_array($b, $c, ); // `strict_param` rule
 foo($a === null, ); // `yoda_style` rule
+$a =(unset)$z; // `no_unset_cast` rule
```
Comment (keradus) [inline tests/Fixtures/Integration/misc/PHP7_3.test]: didn't you pointed out it's pointless to add this rule here ?

Comment (SpacePossum) [reply tests/Fixtures/Integration/misc/PHP7_3.test]: yeah, but could do no harm to prove that here by test?

**Comment by SpacePossum** [comment]
failing tests are:
- PHPMD is run without any files (since no php files are touched in the PR) 
- timeout when using collecting-coverage (this test is allowed to fail)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**